### PR TITLE
More JAM stuff

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- Sort #includes and their magwell classes in lexicographical order
- Put all calibers in their own included .hpp file (except manpad AT and AA launchers, not sure what to do with those yet.
- add a few new magwell calibers (.38 Special, .357 Mag, .408 Chey Tac, .45 Gap, 6.5 Creedmore, 6.5 Grendal)
- add magwells for Glock pistols.
- add 9x18mm Makarov calibre magwells as versions of the same weapons in other calibers.

### CODE REVIEW

**Comment by Drofseh** [comment]
6.8 SPC and .300 Blackout have previously been using metric designations (`CBA_68x43_ACR` and `CBA_762x35_STANAG` respectively).

I only noticed this after I'd added imperial in `magwells_300BLK.hpp` and `magwells_68SPC.hpp`.

I've added the two pre-existing metric classnames to those files so nothing gets broken, but I think we should switch to imperial for those calibers as metric is not commonly used for them.

**Comment by robalo** [comment]
762x39mm in STANAG mag ??

**Comment by Drofseh** [comment]
Yup.
https://i.gyazo.com/e2a14060fcf546be454ed40d7f81a1b6.jpg

File: addons/jam/magwells_303B.hpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-    class CBA_303B_LeeEn {};        // Lee Enfield
     class CBA_303B_BREN {};         // BREN gun
+    class CBA_303B_LeeEn {};        // Lee Enfield
```
Comment (dedmen) [inline addons/jam/magwells_303B.hpp]: Just as a side note. We don't really need to shorten names to oblivion. Longer names don't hurt that much

**Comment by dedmen** [comment]
>I've moved the two pre-existing metric classnames to those files so nothing gets broken, but I think we should switch to imperial for those calibers as metric is not commonly used for them.

I think it's enough to mark them with "deprecated, use X instead" for now.

**Comment by robalo** [comment]
Wouldn't worry about deprecation much at this point. It's unreleased. Feel free to clean up extra crud.

**Comment by dedmen** [comment]
People are already starting to implement it

**Comment by Drofseh** [comment]
@commy2 I hadn't quite pushed the last changes to this PR

**Comment by dedmen** [comment]
That's why you mark PR's as WIP :D
But you can take that for hacktoberfest ^^

**Comment by Drofseh** [comment]
lol, I don't have the ability to add labels to CBA.

**Comment by Drofseh** [comment]
More changes in #997

**Comment by dedmen** [comment]
Not labels.
Just name your PR with `[WIP] PR name` and then later remove the wip tag from title

**Comment by Drofseh** [comment]
Ahh, sure, I will do that in future.

**Pull request merged**